java/client/src/org/openqa/selenium/remote/ProtocolHandshake.java

```diff
@@ -76,7 +76,7 @@ public Result createSession(HttpClient client, Command command)
 
         if (result.isPresent()) {
           Result toReturn = result.get();
-          LOG.info(String.format("Detected dialect: %s", toReturn.dialect));
+          LOG.finest(String.format("Detected dialect: %s", toReturn.dialect));
```

Comment:
This is an incorrect change. The dialect spoken is an important part of the handshake and should be communicated to users.